### ISSUE TITLE
Orientation_notebook

### ISSUE THREAD

**Issue by mmrmas** [unpackAI/DL201#6]
I had an issue with the urllib.request.urlretrieve function. It downloaded the sample-cat.jpg file without error messages but stored it as a corrupted file. 
I borrowed the requests.get() code from one of the blocks below to make it work. 
`# Donwload the image by using its url and filename
# One of several methods to donwload an image

# create directory that is visible from browser 
try:
    os.makedirs("/kaggle/working/inferenceImages/")
except:
    print("directory already exists")   
os.chdir("/kaggle/working/inferenceImages/")

# Upload the image 
image_object = requests.get(image_url).content
with open(image_filename, 'wb') as handler:
    handler.write(image_object)
    
#image_data = PILImage.create(image_filename)`

Note that I changed the working directory to make the files visible in the browser
Note that it's not essential to generate the variable _image_data_ , the model will also accept the jpg file name under _image_filename_ . But this may be different for other datatypes, perhaps
The model can now be called by `prediction_label, p, probabilities = image_learner.predict(image_filename)`

**Comment by jamescavanagh** [comment]
Ok, could you please send the url that you are downloading the image from?

**Comment by lenismerino** [comment]
@mmrmas Yes, this issue was created intentionally to give the students a reason to use the issues section, using requests.get is a preferred method to get the image, also the hosting site used to upload the image provides several links and not all of them work with urllib.request.urlretrieve.

![temp](https://user-images.githubusercontent.com/45776323/155939734-4adfa7ec-09cd-4d41-94d3-2cf0f3505364.jpg)

Using the cells below, try a different link and request.get and it should work,

**Comment by mmrmas** [comment]
> 

@jamescavanagh this was the file: `https://github.com/mmrmas/ant/blob/master/orientation-notebook-local.ipynb`

**Comment by jamescavanagh** [comment]
Ok, great, so you did the right thing by creating an issue from the intentionally created error! Well done!
